README.md

```diff
@@ -216,6 +216,10 @@ __Sample of a real Grabbit Job status__
 
 ![alt text](assets/ExampleGrabbitStatus.jpg)
 
+Two loggers are predefined for Grabbit. One for Grabbit Server and the other for Grabbit Client. 
```

Comment:
Its still worth adding I think; but may be under "advanced monitoring" or some heading maybe?